runtime/tests/utils/utils.go

```diff
@@ -192,3 +193,17 @@ func AssertValueSlicesEqual(t testing.TB, inter *interpreter.Interpreter, expect
 
 	return true
 }
+
+func CheckErrorMessage(err error) {
```

Comment:
why not combine this with `require.Error`, since every invocation asserts that err isn't null?

```suggestion
func CheckErrorMessage(t *testing.T, err error) {
	require.Error(t, err)
	_ = err.Error()
```

Reply:
+1
And perhaps rename `CheckErrorMessage` to something like `RequireError`

Reply:
No reason why not, good idea

Reply:
Refactored in [94a0700](https://github.com/onflow/cadence/pull/2034/commits/94a07006da695f97e4833396c806c846e9514a78)